### ISSUE TITLE
Environment

### ISSUE THREAD

**Issue by julesjacobsen** [phenopackets/phenopacket-schema#64]
This was part of the original phenopacket idea, but has not made it into the new schema yet.

**Comment by julesjacobsen** [comment]
Added initial thoughts in commit a8938ab. Environmental exposire is currently just a list of Exposure messages in the Phenopacket.  @cmungall have you got any feedback on this?

https://github.com/phenopackets/phenopacket-schema/blob/a8938ab16cf031df453ad317b1789767ebe944b9/src/main/proto/base.proto#L161-L215

@pnrobinson  also suggested replacing the start - end with a simpler `Interval` message consisting of two timestamps for the start and end.

**Comment by cmungall** [comment]
we should put ECTO as the example ontology, cc @diatomsRcool

do we also want to allow any kind of post-composition (route, substance, severity, duration) or force use of ECTO class? What about direct use of e.g. CHEBI, chEMBL, ...?

> @pnrobinson also suggested replacing the start - end with a simpler Interval message consisting of two timestamps for the start and end.

I like this suggestion, we get reuse, and by composition not inheritance

**Comment by julesjacobsen** [comment]
> do we also want to allow any kind of post-composition (route, substance, severity, duration) or force use of ECTO class? What about direct use of e.g. CHEBI, chEMBL, ...?

Yes, I think that would be good, although I think I remember @pnrobinson had some reservations about meaningfully quantifying severity. 

Adding a `route` looks good, but not sure how ECTO models that. There are some [weird terms](https://www.ebi.ac.uk/ols/ontologies/ecto/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FECTO_7000010&viewMode=All&siblings=false) in there - for example how did this exposure happen? What body site? How severe and what was the duration? Sudden 'exposure to brick' via sight isn't likely to have much of an effect, but a severe and short 'exposure to brick' via the head may well be a primary cause of bleeding, fractures, oedema and possibly death.

**Comment by pnrobinson** [comment]
This is the latest proposal
```

// An individual environmental exposure, observed as either present or absent (negated), with possible onset, modifiers
// and frequency. Recommended ontology for use is ECTO http://purl.obolibrary.org/obo/ecto.owl
message Exposure {
    // Free-text description of the exposure. Note this is not a acceptable place to document/describe the exposure -
    // the type and onset etc... fields should be used for this purpose.
    string description = 1;

    // The primary ontology class which describes the exposure.
    OntologyClass type = 2;

    // Flag to indicate whether the exposure occurred or not. Default is 'false', in other words the exposure has
    // occurred. Therefore it is only required in cases to indicate that the exposure was deemed _not_ to have happened.
    // More formally, this modifier indicates the logical negation of the OntologyClass used in the 'type' field.
    // *CAUTION* It is imperative to check this field for correct interpretation of the exposure!
    bool negated = 3;

    // Mechanism of the exposure e.g. subclasses of
    OntologyClass route = 4;

    // Ontology class describing the characteristics of the exposure
    repeated OntologyClass modifiers = 5;

    // When the exposure occurred
    TimeElement occurrence = 6;

    //
    Quantity quantity = 8;

    // Evidences for how the exposure was determined.
    repeated Evidence evidence = 7;
}
```
Let's close this issue if there are no objections!

**Comment by pnrobinson** [comment]
@cmungall do you have thoughts on the Exposure element? We will be finalizing shortly!

**Comment by julesjacobsen** [comment]
Closing due to lack of modelling
